core/src/main/java/ai/timefold/solver/core/impl/score/director/AbstractScoreDirector.java

```diff
@@ -260,25 +274,30 @@ private void assertInitScoreZeroOrLess() {
     }
 
     @Override
-    public Score_ doAndProcessMove(Move<Solution_> move, boolean assertMoveScoreFromScratch, Consumer<Score_> moveProcessor) {
-        if (trackingWorkingSolution) {
+    public void executeMove(Move<Solution_> move) {
```

Comment:
Is it safe to skip the assertions here?

Reply:
Yes. When assertions are requested, a different method is called instead.